x-pack/legacy/plugins/graph/public/components/graph_visualization/_graph_visualization.scss

```diff
@@ -1,3 +1,14 @@
+.gphVisualization {
```

Comment:
Making sure the growing flex property is propagated to the svg element. This became necessary because there is an additional wrapper element (introduced by the ngreact-directive). I wasn't able to remove the element (somehow `replace: true`) doesn't work for ngreact directives, thus I chose this solution. The unnecessary wrappers can be removed together with angular.